assets/css/main.css

```diff
@@ -14,3 +14,5 @@
 @import "sections/form";
 
 @import "pages/home";
+@import "tasks/_global";
```

Comment:
Delete lodash from '_global', and '_about-tasks' :)

Reply:
done